partners/project-03.md

```diff
@@ -0,0 +1,57 @@
+---
+altLangPage: "https://conception.canada.ca/partenaires/project-03.html"
+date: 2020-12-08
+dateModified: 2023-12-08
+description: "Partner project: Help develop a prototype for a new procurement website. We focused on the needs of businesses new to government procurement. We tested with small and medium enterprises not currently registered as suppliers to the Government of Canada. Date: Spring/summer 2019"
+title: "Electronic Procurement Solution"
+---
+<p>Our objective was to make make it easier for Canadian businesses to find and understand government business opportunities and guidance.</p>
+<h2>Date</h2>
+<p>Spring/summer 2019</p>
+<h2>Partner department</h2>
+<ul>
+  <li>Public Services and Procurement Canada (PSPC)</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul class="lst-spcd">
+  <li>the federal government is the largest public buyer of goods and services in Canada, with purchases of approximately $23 billion annually</li>
```

Comment:
Capital 't' at beginning... so "The federal...."
(makes it consistent with the capitalization on the other pages)

Reply:
Fixed.